tutorials/data-science-exploration/index.md

```diff
@@ -0,0 +1,301 @@
+distributed across the years:
+
+    $ bq query "select quantiles(year, 11) \
+      from [data-science-getting-started:preprocessing.meteors]"
+
+    Waiting on bqjob_r1173c8efd6d56896_00000159093f2769_1 ... (0s) Current status:
+    DONE
+    +------+
+    | f0_  |
+    +------+
+    | 1400 |
+    | 1974 |
+    | 1979 |
+    | 1985 |
+    | 1988 |
+    | 1991 |
+    | 1995 |
+    | 1999 |
+    | 2001 |
+    | 2003 |
+    | 2013 |
+    +------+
+
+We see here that the period from 1400 to 1974 has roughly the same number of
+reported meteors as the period from 2003 to 2013, though from 1974 onward, there
+seems to have fairly regular reporting.
+
+Quantiles are useful for getting a quick feel for the distribution of your data
+- try using it to find the different masses of meteors found.
+
+[TODO]:# (At times, though, it's helpful to create visualizations of the data, which we'll explore in the [notebooks] and [dashboards] tutorials.)
+
+Finally, one might be concerned, when researching a new place to live or
+establish a business, the rate you might expect meteors to land in your area.
+Fortunately, BigQuery provides some functions to help compute distances between
+latitude and logitude coordinates. Adapted from the [advanced
+examples](/bigquery/docs/reference/legacy-sql#math-adv-examples) in the docs, we
+can find the number of meteors within an approximately 50-mile radius of
+Google's Kirkland campus (at 47.669861, -122.197355):
+
+    $ bq query "select fall, year, mass, name, class, distance_miles \
+      from (select *, \
+       ((ACOS(SIN(47.669861 * PI() / 180) * \
+              SIN(latitude  * PI() / 180) + \
+              COS(47.669861 * PI() / 180) * \
+              COS(latitude  * PI() / 180) * \
+              COS((-122.197355 - longitude) * PI() / 180)) * \
+         180 / PI()) * \
+        60 * 1.1508) as distance_miles \
+       from [data-science-getting-started:preprocessing.meteors]) \
+      where distance_miles < 50"
+
+    Waiting on bqjob_rcabb65ad2a85268_000001590993e5f7_1 ... (0s) Current status: DONE
+    +-------+------+------+--------+-------------------+--------------------+
+    | fall  | year | mass |  name  |       class       |   distance_miles   |
+    +-------+------+------+--------+-------------------+--------------------+
+    | Found | 1925 | 16.7 | Tacoma | Iron, IAB complex | 30.745345223426092 |
+    +-------+------+------+--------+-------------------+--------------------+
+
+Since the last meteor was found before I was born, 30 miles away, it seems that
+Kirkland is relatively safe from meteor landings.
+
+The natural next step in making this information useful and presentable might be
+to create a visualization of this on a map, since this is inherently geographic
+data. For more on this, check out [Cloud Datalab](/datalab) and [Datastudio](//www.google.com/analytics/data-studio/)
+
+[TODO]:# (Head over to the [notebooks] or [dashboards] tutorials for more on this.)
```

Comment:
Future work item - I left them inline so I can remember where there's appropriate places to plug them :-)